<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Publications</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Meena Choi</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="publications.html">Publications</a>
</li>
<li>
  <a href="presentations.html">Presentations</a>
</li>
<li>
  <a href="teaching.html">Teaching</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<ol style="list-style-type: decimal">
<li><p>E. Caron, R. Roncagalli, T. Hase, W. Wolski, T. Hase, <strong>M. Choi</strong>, M. G. Menoita, S. Durand, A. Garcia-Blesa, I. Fierro-Monti, T. Sajic, M. Heusel, T. Weiss, M. Malissen, R. Schlapbach, B. Collins, S. Ghosh, H. Kitano, R. Aebersold, B. Malissen, M. Gstaiger. “Precise temporal profiling of GRB2 protein complexes in primary T cells using SWATH mass spectrometry”. <em>Cell Reports</em>, <a href="DOI:http://dx.doi.org/10.1016/j.celrep.2017.03.019" class="uri">DOI:http://dx.doi.org/10.1016/j.celrep.2017.03.019</a>, 2017.</p></li>
<li><p>S.E. Abbatiello, B.L. Ackermann, C.H. Borchers, R.A. Bradshaw, S.A. Carr, R.J. Chalkley, <strong>M. Choi</strong>, E.W. Deutsch, B. Domon, A.N. Hoofnagle, H. Keshishian, E. Kuhn, D.C. Liebler, M.J. MacCoss, B. MacLean, D.R. Mani, H. Neubert, D. Smith, O. Vitek, L. Zimmerman. “New Guidelines for Publication of Manuscripts Describing Development and Application of Targeted Mass Spectrometry Measurements of Peptides and Proteins”, <em>Molecular &amp; Cellular Proteomics</em>, <a href="doi:10.1074/mcp.E117.067801" class="uri">doi:10.1074/mcp.E117.067801</a>, 2017.</p></li>
<li><p><strong>M. Choi<sup>#</sup></strong>, Z. F. Eren-Dogu<sup>#</sup>, C. Colangelo, J. Cottrell, M. R. Hoopmann, E. A. Kapp, S. Kim, H. Lam, T. A. Neubert, M. Palmblad, B. S. Phinney, S. T. Weintraub, B. MacLean, O. Vitek. “ABRF Proteome Informatics Research Group (iPRG) 2015 Study: Detection of differentially abundant proteins in label-free quantitative LC-MS/MS experiments”. <em>Journal of Proteome Research</em>, <a href="doi:10.1021/acs.jproteome.6b00881" class="uri">doi:10.1021/acs.jproteome.6b00881</a>, 2017.</p></li>
<li><p>C. Terfve, E. Sabidó, Y.Wu, E.Gonc¸alves, <strong>M. Choi</strong>, S. Vaga, O. Vitek, J. Saez-Rodriguez, R. Aebersold. “Systemwide quantitative proteomics of the metabolic syndrome in mice: genotypic and dietary effects”. <em>Journal of Proteome Research</em>, <a href="doi:10.1021/acs.jproteome.6b00815" class="uri">doi:10.1021/acs.jproteome.6b00815</a>, 2016.</p></li>
<li><p>E. Borràs, E. Cantó, <strong>M. Choi</strong>, L. M. Villar, J.C. Álvarez-Cermenño, C. Chiva, X. Montalban, O. Vitek, M. Comabella, E. Sabidó. “Protein-based classifier to predict conversion from Clinically Isolated Syndrome to Multiple Sclerosis”. <em>Molecular &amp; Cellular Proteomics</em>, <a href="doi:10.1074/mcp.M115.053256" class="uri">doi:10.1074/mcp.M115.053256</a>, 2015.</p></li>
<li>S. Surinova, <strong>M. Choi</strong>, S. Tao, P. Schüffler, C. Chang, T. Clough, K. Vysloužil, M. Khoylou, J. Srovnal, Y. Liu, M. Matondo, R.Hüttenhain, H.Weisser, J. M. Buhmann, M. Hajdúch, H. Brenner, O. Vitek, R. Aebersold. “Prediction of colorectal cancer diagnosis based on circulating plasma protein”. <em>EMBO Molecular Medicine</em>, 7(9), 1166-1178,</li>
<li></li>
<li><p>S. Surinova, L. Radová, <strong>M. Choi</strong>, J. Srovnal, H. Brenner, O. Vitek, M. Hajdúch, R. Aebersold, “Non-invasive prognostic protein biomarker signatures associated with colorectal cancer”. <em>EMBO Molecular Medicine</em>, 7(9), 1153-1165, 2015.</p></li>
<li><p><strong>M. Choi</strong>, C. Chang, T. Clough, D. Broudy, T. Killeen, B. MacLean, O. Vitek. “MSstats: an R package for statistical analysis of quantitative mass spectrometry-based proteomic experiments”. <em>Bioinformatics</em>, 30:2524-2536, 2014.</p></li>
<li><p>D. Broudy, T. Killeen, <strong>M. Choi</strong>, N. Shulman, D. Mani, S. Abbatiello, D. Mani, R. Ahmad, A. Sahu, B. Schilling, K.Tamura, Y. Boss, V. Sharma, B. Gibson, S. Carr, O. Vitek, M. MacCoss, B. MacLean. “A framework for installable external tools in Skyline”. <em>Bioinformatics</em>. 30:2521-2523, 2014.</p></li>
<li><p>F. Cerciello, <strong>M. Choi</strong>, A. Nicastri, D. Bausch-Fluck, A. Ziegler, O. Vitek, E. Felley-Bosco, R. Stahel, R. Aebersold, B.Wollscheid. “Identifcation of a seven glycopeptide signature for malignant pleural mesothelioma in human serum by selected reaction monitoring”. <em>Clinical Proteomics</em>, 10:16, 2013.</p></li>
</ol>
<p>For all papers: <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Meena+Choi">Link to PubMed</a>, <a href="https://scholar.google.com/citations?user=S9ccf_oAAAAJ&amp;hl=en">Google scholar</a></p>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
